test/e2e/charts/BUILD.bazel

```diff
@@ -0,0 +1,13 @@
+load("@io_bazel_rules_docker//container:bundle.bzl", "container_bundle")
+
+container_bundle(
+    name = "images",
+    images = {
+        # A set of images to bundle up into a single tarball.
+        "pebble:bazel": "//test/e2e/charts/pebble:image",
+        "quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.21.0": "@io_kubernetes_ingress-nginx//image",
```

Comment:
The nginx-ingress helm chart uses `semverCompare` on the image.tag value, so we cannot use the 'bazel' image tag for nginx like with the other images.